### DISCUSSION TITLE
Add Tailwind installation Guide for React Router v7

### DISCUSSION THREAD

**Discussion by El-Rico** [tailwindlabs/tailwindcss#15313] [Ideas]
Is it possible to add a Tailwind installation guide for the new React Router v7 framework? Or is it advised to follow the Remix installation guide for RR7?

**Comment by matagaralph** [comment]
> Is it possible to add a Tailwind installation guide for the new React Router v7 framework? Or is it advised to follow the Remix installation guide for RR7?

Are you facing issues? It's easy to install in React Router v7. In fact the template comes with tailwindcss installer by default.

**Reply by El-Rico** [reply]
You are completely right.
